jaxrs/resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/ResteasyClientBuilder.java

```diff
@@ -335,7 +336,7 @@ protected ResteasyProviderFactory getProviderFactory()
       if (providerFactory == null)
       {
          // create a new one
-         providerFactory = new ResteasyProviderFactory();

```

Comment:
I thought that made sense, but then I overrided getRuntimeType() and realized that the ResteasyClientBuilder's config should be a LocalResteasyProviderFactory. In fact, I changed withConfig() for that reason.